tests/integration/test_ethereum_tester.py

```diff
@@ -298,6 +298,9 @@ def test_eth_chainId(self, web3):
         assert is_integer(chain_id)
         assert chain_id == 61
 
+    @pytest.mark.xfail(raises=KeyError, reason="ethereum tester doesn't return 'to' key")
+    def test_eth_getTransactionReceipt_mined(self, eth_tester, web3, unlocked_account):
+        super().test_eth_getTransactionReceipt_unmined(web3, unlocked_account)
```

Comment:
@rajeev29five this line should also be `test_eth_getTransactionReceipt_mined`. I think then you'll be in good shape as soon as you fix the lint tests.

Reply:
@kclowes I was about to ask you about that line. I will make the changes test for any other failures. Thanks!

Reply:
@kclowes What should I pass for positional argument: 'mined_txn_hash' ?

Reply:
I think it should be `super().test_eth_getTransactionReceipt_mined(web3, unlocked_account, mined_txn_hash)` like this. Right?

Reply:
You'll need to pass in `web3`, `block_with_txn`, and `mined_txn_hash`, just like in the original definition in `module_testing/eth_module.py`. So the function should look like: 
```
@pytest.mark.fail(...)
def test_eth_getTransactionReceipt_mined(self, web3, block_with_txn, and mined_txn_hash):
        super().test_eth_getTransactionReceipt_mined(web3, block_with_txn, and mined_txn_hash)
```
Sorry for all the confusion. Let me know if you want a better explanation of why.